README.md

```diff
@@ -6,7 +6,7 @@
 
   - https://github.com/iron-people-fec/product_images_proxy
```

Comment:
i think these should point to our service repos and not the proxies, since you need each of them to run the proxy

Reply:
That makes sense. I will have it changed by the next PR